### PR TITLE
Add CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by maclover7**
cc @rafaelfranca

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+#### Push
+
+```
+git push origin my-feature-branch
+```
+
+#### Make a Pull Request
+
+Go to https://github.com/contributor/rails-html-sanitizer and select your feature branch. Click the 'Pull Request' button and fill out the form. Pull requests are usually reviewed within a few days.
```
Comment (rafaelfranca) [inline CONTRIBUTING.md]: This URL may confuse people? Maybe `Go to your fork in GitHub`?

**Comment by rafaelfranca** [comment]
Awesome! I have one minor comment after that we can merge this one.

File: CONTRIBUTING.md (hunk 1 above)
Comment (maclover7) [inline CONTRIBUTING.md]: Prefer this URL - makes it easy so people just slide their GH username in where it says contributor. Not true for all projects, but the name of this project (Rails HTML Santizers) doesn't match the GH project url slug (rails-html-sanitizer). Also, worth updating all of the PRs for this?

Comment (rafaelfranca) [inline CONTRIBUTING.md]: > Also, worth updating all of the PRs for this?

Yeah, it is not.

**Comment by maclover7** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Pull request merged**